roles/signals.py

```diff
@@ -45,6 +45,3 @@ def add_creator_permission(sender, **kwargs):
         instance,
         GroupTypes.REPO_ADMINISTRATOR,
     )
-    # This signal is only fired upon initial creation of a repository,
-    # so receivers don't have to check whether the repository is new.
-    repo_created.send(sender=instance)
```

Comment:
Why is the signal removed?  Was it created only for the curation status vocab?  It sounds like a useful signal though I suppose we don't need to add signals that we don't use.